### ISSUE TITLE
Dodgy HOME metalink (SF issue)

### ISSUE THREAD

**Issue by tanabi** [fuzzball-muck/fuzzball#627]
On a muck with the 'gohome' command set (where gohome
is the usual metalink global exit to *HOME*) do the
following.
```
@crea gh_toy
@crea ride_toy
@link ride_toy=gh_toy
@set ride_toy=x
@flock ride_toy=me
drop ride_toy
drop gh_toy
look
@force ride_toy=gohome
look
```
After the first look you'll see the ride_toy and gh_toy
in the room with you. After forcing the ride_toy to
gohome, you will find that the ride_toy is still in the
room with you while the gh_toy has relocated to #0. A
quick @tel later and you're sitting in #0. This will
work for mortals and wizards. Neither ride_toy nor
gh_toy are linked to #0.

**Comment by PointsDragon** [comment]
@tanabi Wyld and I couldn't replicate this behavior.  Instead, the object simply doesn't get moved, no matter where the parent and child are placed, or in relation to each other... mind you, I'm not sure THIS is the correct behavior either. :)
@wyld-sw so do we close this as not repro, or do we update it to reflect the behavior we're seeing, and determine what SHOULD be happening in this case?  This does appear to have some relation to #628 at this point.

**Comment by wyld-sw** [comment]
Agreed that the issue as written is not reproducible.

However, forcing ride_toy to "home" instead of "gohome" DOES send it into gh_toy, in my testing. I didn't expect #-3 to be THAT different from gohome, but maybe that's expected. If it isn't, then we may want to update this to reflect the discovered issue, assuming THAT's reproducible.

Once we determine what the behavior should be, we can fix or close!

**Comment by tanabi** [comment]
@wyld-sw @PointsDragon these "(SF)" issues aren't anything I really reported, they're things I copy/pasted from Source Forge. So it's likely a lot of these are wild goose chases. I'm closing this because I don't have any additional useful information on this, it's probably not an issue anymore.

**Comment by PointsDragon** [comment]
Ah, I was hoping you might have some additional insight - because we did find some additional issues, just not THIS issue as written.  Hitting an exit linked to HOME should have teleported that object into the other object, not a 'successful' trigger of the action followed by nothing.  @wyld-sw guessing we should just open a fresh issue on this one, perhaps.

**Comment by tanabi** [comment]
@PointsDragon Yeah, sadly I don't know anything about any issue that  has (SF issue) in the title. Wyld and I debated whether we should even bother 'porting' them over and ultimately we decided it makes more sense to be thorough than to let something potentially important drop. So I did the work to copy/paste the issues over, so they are all in my name, but I don't have any details that aren't in the ticket and some of them are kinda shoddy (I did weed out the shoddiest ones at least!).

We might be able to backtrack to the original sourceforge issue to at least find the ticket author, but would they remember something they reported 15+ years ago? ... or even still be around? :) I don't know. Probably not worth chasing..
